TypeInjections/Tool/Tool.fs

```diff
@@ -4,11 +4,13 @@
 namespace Tool
 
 open System.Collections.Generic
+open System.Text.RegularExpressions
```

Comment:
You should not do any regex handling at all. You should leave parsing of ignore files to the API.

Reply:
Yeah this was me trying to simplify the tool for the user. I wanted the CLI to accept this input
`$ typecart local -o [OldPath] -n [NewPath] --ignore file1 file4`
And the typeCart API had this option `new(d: DirectoryInfo, ignorePatterns: Regex list)` so I took the users string input and cast it to `Regex`. 

Next PR I wanted to add something like `--typecartignore` where the user can input a `.typecartingores file`

Reply:
You shouldn't turn the list of strings into regexes. We process the strings in a special way so that they are easier to write. See the README for details.

Reply:
Just pass in the list of strings will suffice. I would also prefer if you add the option of supplying a file in the same PR. This is the current way we're using typecart, and it would be nice if we have the same user interface in the cli tool as well

Reply:
Yeah I liked your `makeRegex` method and wanted to use it. However, I don't see an option to pass in `string list`. I could only find `string option` for inputting a path to `.typcartignore` and `Regex list`

Reply:
added --typecartignore